### DISCUSSION TITLE
215

### DISCUSSION THREAD

**Discussion by giscus** [Shane-Park/markdownBlog#175] [Comments]
# 215

2022년 1월 1일 기준으로 새로 업데이트 합니다. 인텔리제이 버전이 2021.3으로 업데이트 되며 메뉴 구성이 약간씩 바뀌어서 거기에 맞춰 새로 변화 했습니다. 설정이 자주 변하다보니 인터넷에서 정보를 찾는 입장에서는 많이 불편 할 수 있지만 자세히 보면 결국 위치만 바뀐게 대부분이라서 최대한 변화에 맞춰 글을 업데이트 하는 방향으로 해보겠습니다. 혹시 버전이 바뀌어 메뉴가 또 일치하지 않는 경우가 있다면 댓글로 달아주시면 바로 반영 하도록 하겠습니다. 아래의 문서를 참고 했습니다. https://docs.spring.io/spring-boot/docs/current/reference/html/using.html#using.devtools Intro 스프링 부트 뿐만 아니라 웹 프로젝트를 할 때에 단..

https://shanepark.tistory.com/215

**Comment by HyungKuIm** [comment]
컴퓨터 속도가 좀 빨라야 되나봐요. 편리하긴 한데 컴퓨터 팬이 너무 도네요. 잘 정리된 글 감사합니다.
